packages/@aws-cdk/aws-ecs/lib/container-image.ts

```diff
@@ -1,17 +1,28 @@
 import ecr = require('@aws-cdk/aws-ecr');
+import secretsmanager = require('@aws-cdk/aws-secretsmanager');
 import cdk = require('@aws-cdk/cdk');
-
 import { ContainerDefinition } from './container-definition';
+import { CfnTaskDefinition } from './ecs.generated';
+
+/**
+ * Repository Credential resources
+ */
+export interface RepositoryCreds {
```

Comment:
What are your thoughts for this type? Future proofing? Because it's not strictly necessary today, right?

Reply:
Correct, today we could potentially add a secret to ContainerImage directly but I want to minimize the need for rewiring if other resources or additional fields become valid in the future. Do you foresee issues with this approach?